HISTORY.rst

```diff
@@ -13,6 +13,10 @@ Other Changes
 * Warnings are now emitted if the user attempts to run the regridding utilities with a version of `xarray` that is not compatible with `cf-xarray`. (#310).
 * Dependency pins now constrain the `xarray` version when installing with `$ pip install ".[extra]"`. (#310).
 
+New Features
+^^^^^^^^^^^^
+* ``clisops.ops.average.average_shape`` added (#XXX). Exposing average_shape from clisops.core to clisops.ops.
+
```

Comment:
@charlesgauthier-udm This should be placed in a new section for `v0.12.2 (unreleased)`

Reply:
Will do and will add PR number as well